packages/paste-core/components/label/src/Label.tsx

```diff
@@ -64,26 +69,21 @@ const Label = React.forwardRef<HTMLLabelElement, LabelProps>(
         paddingLeft="space0"
         paddingRight="space0"
         textTransform="none"
+        element={element}
+        fontSize="fontSize30"
+        fontWeight="fontWeightBold"
+        lineHeight="lineHeight30"
+        color={textColor}
+        cursor={disabled ? 'not-allowed' : 'pointer'}
         ref={ref}
       >
         <MediaObject verticalAlign="top">
           {required && (
             <MediaFigure spacing="space20">
-              <RequiredDot />
+              <RequiredDot element={element} />
```

Comment:
I'm not sure I know why I prefer this, but I think I would treat Required Dot as a stand alone component that deals with setting it's own element name.

Then here, because it's a composed use case, I might provide a composed element name that overrides the default. `element={`${element}_REQUIRED_DOT`}`.

That way if I were to use Required Dot on it's own, as it is exported so you can, I would get an element name by default.

```
<RequiredDot />
```

would result in `data-paste-element="REQUIRED_DOT"`. Right now I would get `data-paste-element="_REQUIRED_DOT"`, because element is not provided.